src/ch/imedias/rsccfx/view/RsccShowTokenView.java

```diff
@@ -120,12 +115,16 @@ private void bindFieldsToModel() {
     pane.setCenter(boxCenter);
     pane.setBottom(boxBottom);
     this.getChildren().add(pane);
+
+    tf.textProperty().bind(model.keyProperty());
   }
 
-  /** initSize method. */
+  /** initSize method. */ // FIXME: format javadoc comment according to the google java code style guidelines
```

Comment:
@Gyssim format javadoc comment according to the google java code style guidelines